app/controllers/books_controller.rb

```diff
@@ -8,4 +8,9 @@ def index
   def show
     render json: Book.find(params[:id]), serializer: BookShowSerializer
   end
+
+  def find_by_isbn
+    book_service = OpenLibraryService.new
```

Comment:
if the instance doesn't change in every instance you could use it as a singleton
`book_service = open_library_service`

```
def open_library_service
  @open_library_service ||= OpenLibraryService.new
end
```